fiatconnect-api.md

```diff
@@ -1544,8 +1538,8 @@ If a transfer is not allowed for a generic reason (such as unacceptable transfer
 
 ###### 3.3.4.1.3.2.3. `InvalidQuote`
 
-If the quote associated with `quoteId` is expired, or if no quote is found for the user with a matching `quoteId`, the server MUST
-reject the transfer and return an `InvalidQuote` error. 
+If the quote associated with `quoteId` is expired, or if no quote is found for the user with a matching `quoteId`, or if the provided quote is for the
```

Comment:
nit: might make sense to update the "semantics" section to indicate that not only the exchange rate and fees, but also the amount, cryptoType, and fiatType, should be used from the original quote

Reply:
relatedly, this sentence from `3.3.1.2.3.1` may need to be updated:
```
The `quote.quoteId` field
is a globally unique identifier for the quote, and is used by the client to initiate a transfer at the
conversion rate and fee amount given by the quote.
```